### ISSUE TITLE
Viewing hint for non-paged Canvases

### ISSUE THREAD

**Issue by mikeapp** [yalelibrary/YUL-DC#142]
**Issue**

In paged objects there are sometimes additional images inserted into the sequence.  For instance, in some cases a page is shot twice.   The additional image causes the two-up book viewer mode to not show facing pages.  See the example below.

The solution is to provide a viewing hint (or behavior in IIIF Presentation 3) on the IIIF Canvas that indicates that the Canvas is not part of the page sequence.

 ![Screen Shot 2020-05-13 at 9.53.46 PM.png](https://images.zenhubusercontent.com/5e5d472f410278efd81466ab/8d623995-1be9-4c19-aaf8-2a7333ad62da)

**Acceptance**

- [ ] Image-level viewingHint from Goobi is expressed in the corresponding IIIF Canvas

Note:  This field should be coming in the METS data.  If not, we need to create a new ticket to make sure we're getting the data we need.

**Comment by mikeapp** [comment]
This might be done already?

**Comment by alishaevn** [comment]
@mikeapp  the manifest in the description doesn't currently have a 2 page view available. so maybe this ticket _has_ been solved? if I remember correctly, isn't it the manifest that determines if the uv will show the 2 page option?

http://universalviewer.io/uv.html?manifest=https://collections-uat.library.yale.edu/manifests/10958722#?c=0&m=0&s=0&cv=4&xywh=-1249%2C226%2C3433%2C2908

![image.png](https://images.zenhubusercontent.com/5e5ea9bffa78052c1cbcc74f/ab34648a-58ad-414b-994d-bdbf39a1d1e8)

**Comment by mikeapp** [comment]
At the Manifest (/object) level, there has to be a `viewingHint` (https://iiif.io/api/presentation/2.1/#viewinghint) of `paged` to trigger the two-up book reader view.

At the Canvas (/child) level, there are additional `viewingHint`s  that can be applied, see the drop-down:

![Screen Shot 2021-04-20 at 6 01 13 PM](https://user-images.githubusercontent.com/1987952/115469341-787ca100-a202-11eb-9bf4-2e03b1773c83.png)

These should be expressed on that child's Canvas, for example if the first Canvas were `non-paged` it would look like:

```
{
  "@context": "http://iiif.io/api/presentation/2/context.json",
  "@id": "https://collections-uat.library.yale.edu/manifests/10958722",
  "@type": "sc:Manifest",
  "label": "Texts on St. Jerome",
  "sequences": [
    {
      "@id": "https://collections-uat.library.yale.edu/manifests/sequence/10958722",
      "@type": "sc:Sequence",
      "canvases": [
        {
          "@id": "https://collections-uat.library.yale.edu/manifests/oid/10958722/canvas/10962124",
          "@type": "sc:Canvas",
          "viewingHint": "non-paged",
          "images": [
            {
              "@id": "https://collections-uat.library.yale.edu/manifests/oid/10958722/canvas/10962124/image/1",
              "@type": "oa:Annotation",
              "motivation": "sc:painting",
etc....
```

There's no guarantee that the viewer will actually render this correctly, so just focus on outputting the `viewingHint` in the Manifest JSON correctly.

**Comment by dylansalay** [comment]
We uncovered an issue where the parent object was not re-creating the manifest when it was updated. We addressed that in this ticket so now when a viewing hint is added, the parent object will reflect that viewing hint in the manifest and the universal viewer. 

We will be making a new ticket for re-creating the manifest when a child object is updated. For now, if you need to update the viewing hint for a child object, you will need to update the viewing hint on the child object, and then update the parent object so the manifest is regenerated.

**Comment by alishaevn** [comment]
The work for this ticket is complete, but we need to address the failing specs and drop in coveralls coverage.

**Comment by FCRodriguez7** [comment]
Needs discussion for rerunning manifest.

**Comment by FCRodriguez7** [comment]
Released management v2.36.1

**Comment by mikeapp** [comment]
Looks good on test.
